### DISCUSSION TITLE
Is there a bot.reply_to() function, but for sending files?

### DISCUSSION THREAD

**Discussion by Zapzatron** [eternnoir/pyTelegramBotAPI#2301] [Q&A] [answered]
I need to be able to call the function that is responsible for replying to a message when sending a document, just like when sending text. 
For example, there is bot.reply_to(), which replaces bot.send_message(). Is there a function to replace bot.send_document()?
And are there plans to add it?

**Comment by Badiboy** [comment]
reply_to is a syntactic sugar for frequently used function.

You can do replies with send_xxx by setting reply_to_message_id parameter.

**Reply by Zapzatron** [reply]
I'm using it now:
```python
await bot.send_document(req_message[“chat_id”], document, message.message_id)
```

But I get the warning `The parameter 'reply_to_message_id' is deprecated. Use 'reply_parameters' instead.`.
It would be cool to have a function like reply_to, so that you can keep track of parameter changes without having to track them.

**Reply by coder2020official** [reply, marked as answer]
Yep, use reply_parameters instead, check the documentation

**Reply by Badiboy** [reply]
Thus reply_parameters. I forgot that Telegram changed the signature.

**Reply by Zapzatron** [reply]
Using ReplyParameters is not a problem, I just wanted to clarify if a function like reply_to() is planned.

**Reply by coder2020official** [reply]
Implementing another reply_to would be useless imho. Adding reply_tos to every content_type is even more ridiculous.
I previously had an idea to add reply() functions and other types of "quick" functions to classes such as Message; Though we then rejected the idea as apparently, on a closer look, it will create a mess in the types file;

**Reply by Zapzatron** [reply]
Got it, thanks for your work.
